data.sql

```diff
@@ -10,4 +10,4 @@ INSERT INTO animals.animals VALUES (7, 'Squirtle', '1993-04-02', 3, false, 12.13
 INSERT INTO animals.animals VALUES (8, 'Angemon', '2005-06-12', 1, true, 45)
 INSERT INTO animals.animals VALUES (9, 'Boamon', '2005-06-07', 7, true, 20.4)
 INSERT INTO animals.animals VALUES (10, 'Blossom', '1998-10-13', 3, true, 17)
-INSERT INTO animals.animals VALUES (11, 'Ditto', '2022-05-14', 4, true, 22)
\ No newline at end of file
+INSERT INTO animals.animals VALUES (11, 'Ditto', '2022-05-14', 4, true, 22)
```

Comment:
- [ ] Please follow the given guidelines to update and delete records currently I am not able to see changes in your repo please address that